metricsaggregator/src/main/java/io/dockstore/metricsaggregator/client/cli/CommandLineArgs.java

```diff
@@ -37,29 +37,29 @@ public static class AggregateMetricsCommand extends CommandLineArgs {
         @Parameter(names = {"-c", "--config"}, description = "The config file path.")
         private File config = new File("./" + MetricsAggregatorClient.CONFIG_FILE_NAME);
 
-        @Parameter(names = {"--athena"}, description = "Aggregate metrics in S3 using AWS Athena")
-        private boolean withAthena = false;
+        @Parameter(names = {"--trsIds"}, description = "Aggregate metrics for the tools specified by their TRS IDs")
+        private List<String> trsIds;
 
-        @Parameter(names = {"--skipDockstore"}, description = "Skip posting the metrics to Dockstore")
-        private boolean skipDockstore = false;
+        @Parameter(names = {"--allS3"}, description = "Aggregate all executions in S3, even if they have been aggregated before")
```

Comment:
Added this command that aggregates everything in S3. I'm thinking we should do this for the release so that the `latestmetricsaggregationdate` gets set for entries with existing metrics introduced in this PR:

https://github.com/dockstore/dockstore/pull/5970/files#diff-44505a718a3a6f727838e1db36b9a41b769c720b191c4cd74ae542e17d740887